components/hero.js

```diff
@@ -253,6 +253,7 @@ export class Hero extends PureComponent {
               <LoopedTextPart>zebedee.io</LoopedTextPart>
               <LoopedTextPart>coinos.io</LoopedTextPart>
               <LoopedTextPart>ln.tips</LoopedTextPart>
+              <LoopedTextPart>getalby.com</LoopedTextPart>
```

Comment:
So if you connect your node to the Alby browser extension, you automatically get a `getalby.com` domain? I'm not sure that's the case. This piece is to loop through the many domains available.

Reply:
You don't automatically get a domain but with a few steps you can setup your Alby domain.

Reply:
I talked to the maintainers of Alby and they agree that getting a @getalby.com lightning address right now is sort of a hidden feature. You have to go to https://github.com/login/ to do it. So I have removed this part from the PR and we can add it in at a later date when the lightning address is more integrated for users.

Reply:
I talked to the maintainers of Alby and they agree that getting a @getalby.com lightning address right now is sort of a hidden feature. You have to go to https://github.com/login/ to do it. So I have removed this part from the PR and we can add it in at a later date when the lightning address is more integrated for users.